packages/expo-cli/src/commands/export.ts

```diff
@@ -80,49 +80,15 @@ export async function action(projectDir: string, options: Options) {
     console.warn(`Dev Mode: publicUrl ${options.publicUrl} does not conform to HTTP format.`);
   }
 
-  const target = options.target ?? getDefaultTarget(projectDir);
```

Comment:
I don't see any tests for this command, we should probably add at least one e2e test before merging this so we can verify that the change isn't unexpected.